userguide/storage.rst

```diff
@@ -2431,6 +2427,44 @@ The remaining settings are left at the default choices. Make sure
 replication task.
 
 
+.. _One-Time Replication:
+
+One-Time Replication
+^^^^^^^^^^^^^^^^^^^^
+
+One-time replications copy manually created snapshots of a dataset to
+another target dataset. This kind of replication is compatible with any
+replication :guilabel:`Transport` method. To configure a one-time
+replication, go to
+:menuselection:`Storage --> Replication Tasks`
+and click :guilabel:`Add Replication`.
+
+Enter a descriptive :guilabel:`Name`, choose the :guilabel:`Direction`,
+and configure any desired :guilabel:`Transport` settings.
+
+Enter the :guilabel:`Source Datasets` with snapshots to be copied to the
+target.
+
+Enter the :guilabel:`Target Dataset` where replicated snapshots will be
+stored and decide whether child dataset snapshots should also be
+replicated.
+
+Do not select a periodic snapshot task for one-time replications. To
+replicate a single snapshot, enter the name of the snapshot in
+:guilabel:`Also include naming schema`. Replace the year, month, day,
+hour, and minute values with *%Y*, *%m*, *%d*, *%H*, and *%M* strings.
```

Comment:
These should use :literal: tags, since they are literal values.

This seems really confusing for the user, though.  They have to enter a pattern match to duplicate a single snapshot?

Reply:
That does appear to be the current behavior.